rma_delivery/tests/test_rma_delivery.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2022 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests import Form, SavepointCase
+
+
+class TestRma(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.company = cls.env.user.company_id
```

Comment:
No possible to inherit from a RMA base test? Moment to do it?

Reply:
Ok, done :)